### PR TITLE
add a readme for the workbuilder and workapplier

### PR DESCRIPTION
**Pull request by zhiweiyin318**
Signed-off-by: Zhiwei Yin <zyin@redhat.com>

### CODE REVIEW

File: pkg/common/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    There are 2 types to create a `WorkApplier` instance.
+    
+    One is `NewWorkApplierWithTypedClient(workClient workv1client.Interface, workLister worklister.ManifestWorkLister)` 
+    with manifestWork typed client. 
+
+    Another is`NewWorkApplierWithRuntimeClient(workClient client.Client)` with runtime client.
+
+2. Apply a manifestWork.
+    
+    This method will create the manifestWork if the manifestWork is not existed, and will update the existing manifestWork.
```
Comment (qiujian16) [inline pkg/common/README.md]: does not exist

File: pkg/common/README.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+        workbuilder.ManifestWorkExecutorOption(executorOption),
+        workbuilder.DeletionOption(deletionOption))
+    ```
+
+6. Build and apply the manifestWorks.
+
+    The user also can use `BuildAndApply` method to apply the built manifestWorks with a `workApplier` instance.
+
+    ```go
+    applyWorks, deleteWorks, err := workBuilder.BuildAndApply(context, manifestObjects, myGenerateWorkObjectMeta,
```
Comment (qiujian16) [inline pkg/common/README.md]: do still have applyWorks, deleteWorks for this?

File: pkg/common/README.md (hunk 1 above)
Comment (zhiweiyin318) [reply pkg/common/README.md]: fixed

File: pkg/common/README.md (hunk 2 above)
Comment (zhiweiyin318) [reply pkg/common/README.md]: no, my mistake. fixed.

**Pull request merged**